pkg/pipelines/argocd/argocd.go

```diff
@@ -203,10 +209,18 @@ func argoCDResource(ns string) (*argov1.ArgoCD, error) {
 	}, nil
 }
 
-func makeApplication(appName, argoNS, project, ns, server string, source *argoappv1.ApplicationSource) *argoappv1.Application {
+func makeApplication(app *config.Application, appName, argoNS, project, ns, server string, source *argoappv1.ApplicationSource) *argoappv1.Application {
+	options := []meta.ObjectMetaOpt{}
+	if app != nil {
+		options = append(options, meta.AddLabels(map[string]string{
+			appLabel: app.Name,
```

Comment:
Just to confirm if I read this correctly, this will generate a label with this format:

`app.kubernetes.io/name=myAppName`

where from the code, `app.Name` is, as an example, has a value of `myAppName`

Is there any advantage if we use a KAM-specific or OpenShift-specific label for those apps generated by kam?

eg. `app.openshift.io/name` perhaps?

If we are linking to ArgoCD from dev console, do we need to filter based on only these types of apps?

Reply:
`app.kubernetes.io/name` is one of the recommended names, but see here https://github.com/openshift/odo/issues/599 and this https://github.com/gorkem/app-labels/blob/master/labels-annotation-for-openshift.adoc for some more detail WRT this and OpenShift.

Essentially, this is simplifying the number of labels needed, the `app.kubernetes.io/name` label is used by the Topology view to group components too, and adding `app.kubernetes.io/name` and possibly `app.openshift.io/name` would mean that we'd presumably have two labels with the same value?

Reply:
Ok, makes sense.